python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -106,47 +148,24 @@ def mean(self, fiberId=None):
 
     def stdev(self, fiberId=None):
         """Return the standard deviation of wavelength residuals (nm).
-
         Parameters
         ----------
         fiberId : `int`, optional
             Fiber identifier to select.
-
         Returns
         -------
         stdev : `float`
             Standard deviation of wavelength residuals (nm).
         """
         return self.residuals(fiberId).std()
 
-    @classmethod
-    def fromSpectrumSet(cls, spectrumSet, detectorMap):
```

Comment:
Why get rid of this method? It seems like a good, self-contained way of constructing the object.

Reply:
I modified the code so that I can get the info directly from calibrateWavelengthTask (or I should duplicate most of the line) and this method is not used to get the info.
I’ll fix it if needed.